plane/source/docs/quadplane-parameters.rst

```diff
@@ -29,6 +29,37 @@ Key Parameters
 -  The default :ref:`SCHED_LOOP_RATE<SCHED_LOOP_RATE>` for a QuadPlane is to 300 (Hz). Most QuadPlanes do not need this to be rasied. Some very small vehicles (< 1Kg) might benefit from setting it to 400. In heavier vehicles, their higher inertia results in lower effective control response rates, so they do not benefit from a higher loop rate. Raising above 300 only leads to larger log files in these vehicles.
 
+.. _return_behavior_setup:
+
+Return Behavior QuickSetup Guide:
+=================================
+
+While there are many parameters setting distances and heights for the various home return modes/behaviors (read each mode's wiki page), this is a quick setup guide for basic behaviors.
+
+1.If you lose RC link for greater than :ref:`FS_LONG_TIMEOUT<FS_LONG_TIMEOUT>` in fixed wing flight set :ref:`FS_LONG_ACTN<FS_LONG_ACTN>` value below to obtain various behavior options:
+- "0" : do nothing if in AUTO mode, otherwise switch to RTL mode
+- "1" : switch to RTL mode
+- "2" : cut throttle and glide in FWBA mode
+- "3" : deploy parachute (assuming you have one setup)
+- "4" : switch to AUTO mode and execute mission at current mission sequence pointer
+
+2. If you lose RC link for greater than :ref:`FS_LONG_TIMEOUT<FS_LONG_TIMEOUT>` in VTOL flight, you will immediately QLAND, unless you select the following optional behavior:
+- set :ref:`Q_OPTIONS<Q_OPTIONS>` bit 5 to switch to QRTL mode instead. (see #5 below)
+- set :ref:`Q_OPTIONS<Q_OPTIONS>` bit 20 to switch to RTL mode instead. (see #3,4 below)
+
+3. Anytime you switch to RTL (due either to manual mode change or failsafe action), do you want to execute an autoland sequence (does not need to actually have a land command, if some other action is desired), rather than just return and loiter around home?
+- If yes,then set up a DO_LAND_START mission sequence and enable the :ref:`RTL_AUTOLAND<RTL_AUTOLAND>` parameter. See :ref:`do_land_start` for details of setup.
+
+4. If in a VTOL mode, and you switch to RTL (due either to manual mode change or failsafe action), then set the :ref:`Q_RTL_MODE<Q_RTL_MODE>` parameter value as follows to determine the behavior:
+- "0" : Switch to normal RTL mode, transitioning to fixed wing (see #3 above)
+- "1" : Transition to fixed wing, fly towards home, transition back to VTOL mode, move to over home, switch to QLAND and land at home in VTOL.
+- "2" : Transition to fixed wing, and do a special approach to home,  transition back to VTOL mode, move to over home, switch to QLAND and land at home in VTOL
+- "3" :Switch to QRTL mode. See below.
+
+5. When switching to QRTL, instead of transitioning to fixed wing (assuming you are not close to home already), flying back to home, then switching to VTOL mode as you approach home, then switching to QLAND and landing at home (see :ref:`qrtl-mode` for more information), you can disable the fixed wing approach and return home and land in only VTOL mode if :ref:`Q_OPTIONS<Q_OPTIONS>` bit 16 set.
```

Comment:
This seems very convoluted. Try rewording it based on a positive statement .. i.e. don't start with "instead of" ... something like:
The standard QRTL will transition to fixed wing, fly back home, switch to VTOL then switch to QLAND to land at the home location. You can disable the fixed wing approach and return home and land in only VTOL mode if :ref:`Q_OPTIONS<Q_OPTIONS>` bit 16 set.

Reply:
reworded a bit...confused even me with a clearer head